pkg/apis/config/feature_flags.go

```diff
@@ -187,3 +188,29 @@ func setEmbeddedStatus(cfgMap map[string]string, defaultValue string, feature *s
 func NewFeatureFlagsFromConfigMap(config *corev1.ConfigMap) (*FeatureFlags, error) {
 	return NewFeatureFlagsFromMap(config.Data)
 }
+
+// GetContextBasedOnFeatureFlag returns a context based on the feature flag
+func GetContextBasedOnFeatureFlag(featureFlag string) context.Context {
```

Comment:
might be worth stating that these functions are for use in testing

Reply:
Added to EnableAlphaAPIFields function.

Feel like `getContextBasedOnFeatureFlag` is overlapping with `EnableAlphaAPIFields` so I just merged and keep `EnableAlphaAPIFields` only. That looks more clean :)